pluginLauncher/lib/source/DobbyRdkPluginUtils.cpp

```diff
@@ -59,19 +59,27 @@ DobbyRdkPluginUtils::~DobbyRdkPluginUtils()
  *  function only parses the pid from a string.
  *
  *  NOTE: Only works with OCI hooks.
- *  @param[in]  stdin            stdin contents from the context of the hook
  *
- *  @return container pid, 0 if none found
+ *  @param[in]  stdin            stdin contents from the context of the hook.
+ *
+ *  @return container pid, 0 if none found.
  */
 pid_t DobbyRdkPluginUtils::getContainerPid(const std::string &stdin) const
 {
     if (stdin.empty())
     {
+        AI_LOG_ERROR_EXIT("container stdin empty - couldn't get pid");
         return 0;
     }
 
     // get pid from hook's stdin json '"pid":xxxxx'
     std::size_t pidPosition = stdin.find("\"pid\":") + 6;
+    if (pidPosition == std::string::npos)
```

Comment:
If `find()` fails to find pid, we still add 6. `npos` is `-1`, so if the PID isn't found `pidPosition` would equal 5 and this error check would never be called.

The `+ 6` should be moved after this check